e2e/pages/adf/demo-shell/process-services/formDemoPage.ts

```diff
@@ -0,0 +1,44 @@
+/*!
+ * @license
+ * Copyright 2019 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ConfigEditorPage } from '../../configEditorPage';
+import { BrowserActions } from '@alfresco/adf-testing';
+import { by, element, browser, ElementFinder } from 'protractor';
+
+export class FormDemoPage {
+
+    formCloudEditor: ElementFinder = element.all(by.css('.mat-tab-list .mat-tab-label')).get(1);
+    formCloudRender: ElementFinder = element.all(by.css('.mat-tab-list .mat-tab-label')).get(0);
+
+    configEditorPage = new ConfigEditorPage();
+
+    async goToEditor(): Promise<void> {
+        await BrowserActions.click(this.formCloudEditor);
+    }
+
+    async goToRenderedForm(): Promise<void> {
+        await BrowserActions.click(this.formCloudRender);
+    }
+
+    async setConfigToEditor(text): Promise<void> {
+        await this.goToEditor();
+        await browser.sleep(2000);
```

Comment:
maybe you can change this sleep with checking if some elements of that page are present